### PR TITLE
feat added watchlater

### PR DESCRIPTION
**Pull request by AmishMishra11**
- added option to add a video to watch later 
- added  option to remove the video from watch later
- watch later page

https://user-images.githubusercontent.com/76205249/164886523-298fc64b-4885-4ac5-a2f3-019560a6be35.mp4

### CODE REVIEW

**Review by Hardiegogo — APPROVED** [review]
Good code structure ,
clean use of utility functions.

File: src/Components/WatchLater/WatchLater.jsx [hunk 1]
```diff
@@ -1,7 +1,41 @@
 import React from "react";
+import { useVideo } from "../../Context/VideoContext";
```
Comment (anshikaa12) [inline src/Components/WatchLater/WatchLater.jsx:2]: you can create an index.js inside contexts folder and import all the contexts from there!

**Review by anshikaa12 — APPROVED** [review]
code is clean and well structured

File: src/Components/WatchLater/WatchLater.jsx (hunk 1 above)
Comment (AmishMishra11) [reply src/Components/WatchLater/WatchLater.jsx:2]: Thank you for the suggestion. But I only have 2 contexts so I don't think there's a need to create a separate index file just for 2 files.

**Pull request merged**